src/main.jsx

```diff
@@ -4,6 +4,12 @@ import CssBaseline from '@mui/material/CssBaseline';
 import { ThemeProvider } from '@mui/material/styles';
 import App from './App';
 import theme from './theme';
+import ReactGA from 'react-ga';
+
+if (navigator.doNotTrack !== "1") {
+  ReactGA.initialize('G-B9WH7S75Z2');
```

Comment:
Not great that we hard-code the key, but I haven't yet figured out how we can include secrets in a static site.